dali/test/python/test_dali_cpu_only.py

```diff
@@ -341,35 +337,39 @@ def test_multi_paste_cpu():
 
 
 def test_roi_random_crop_cpu():
-    check_single_input(fn.roi_random_crop, crop_shape=[x // 2 for x in test_data_shape], roi_start=[x // 4 for x in test_data_shape], roi_shape=[x // 2 for x in test_data_shape])
+    check_single_input(fn.roi_random_crop,
+                       crop_shape=[x // 2 for x in test_data_shape],
+                       roi_start=[x // 4 for x in test_data_shape],
+                       roi_shape=[x // 2 for x in test_data_shape])
 
 
 def test_random_object_bbox_cpu():
     get_data = [
         np.int32([[1, 0, 0, 0],
-                [1, 2, 2, 1],
-                [1, 1, 2, 0],
-                [2, 0, 0, 1]]),
+                  [1, 2, 2, 1],
+                  [1, 1, 2, 0],
+                  [2, 0, 0, 1]]),
 
         np.int32([[0, 3, 3, 0],
-                [1, 0, 1, 2],
-                [0, 1, 1, 0],
-                [0, 2, 0, 1],
-                [0, 2, 2, 1]])
+                  [1, 0, 1, 2],
+                  [0, 1, 1, 0],
+                  [0, 2, 0, 1],
+                  [0, 2, 2, 1]])
     ]
-    check_single_input(fn.segmentation.random_object_bbox, get_data=get_data, batch=False, cycle="quiet", input_layout="")
+    check_single_input(fn.segmentation.random_object_bbox, get_data=get_data, batch=False,
+                       cycle="quiet", input_layout="")
 
 
 def test_numba_func_cpu():
     def set_all_values_to_255_batch(out0, in0):
         out0[0][:] = 255
 
     def setup_out_shape(out_shape, in_shape):
-        out_shape = in_shape
+        out_shape = in_shape  # noqa: F841
```

Comment:
It may be out of scope, but this assignment does seem to do nothing indeed. I wonder if the intention was more like out_shape[i] = in_shape[i]

Reply:
Changed to `pass`, as this is a no-op statement.